script/generate-release-notes.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+PACKAGE_VERSION=$(cat package.json \
+  | grep version \
+  | head -1 \
+  | awk -F: '{ print $2 }' \
+  | sed 's/[",]//g')
+
+dot_version=`echo $PACKAGE_VERSION | sed -E "s/-rc.*$//"`
+dash_version=`echo ${dot_version} | tr '.' '_'`
+
+RELEASE_NOTES_FILE="release_notes/v${dash_version}.html.md"
+
+renogen --format markdown v${dot_version} >> ${RELEASE_NOTES_FILE}
```

Comment:
We'll need a Gemfile/Gemfile.lock to ensure `renogen` is present

Reply:
Do we as its installed globally on a machine. Or can be. Didn't want to cruft up the repo

Reply:
Although if we want the check as part of build we will probably need it

Reply:
Global install is fine, we need instructions for that somewhere though right?